packages/google_sign_in/google_sign_in/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 * Add iOS unit and UI integration test targets.
 * Add iOS unit test module map.
 * Exclude arm64 simulators in example app.
+* Remove references to the Android V1 embedding.
```

Comment:
You need to change the version, not add to an already-published version's release notes.

Reply:
sorry. fixed.